src/components/projects/Projects.css

```diff
@@ -2,6 +2,33 @@
   border: 1px solid red;
 } */
 
+/* LOADING STYLES */
+
+@keyframes shimmer {
```

Comment:
Cool animation!

Reply:
I liked it too
I wonder if the one you were talking about, jake, just has some skinny horizontal containers to make it look like text

Reply:
Me too! I like those text bars, I'll work on that!